manticore/core/smtlib/solver.py

```diff
@@ -129,7 +134,7 @@ def __init__(self):
         super().__init__()
         self._proc = None
 
-        self._command = 'z3 -t:240000 -memory:16384 -smt2 -in'
+        self._command = f'z3 -t:{consts.timeout*1000} -memory:{consts.memory} -smt2 -in'
```

Comment:
We could add config parameter for z3 path and default it to `z3` aka fetch from $PATH.